CMakeLists.txt

```diff
@@ -477,6 +477,18 @@ include_directories(SYSTEM velox/external)
 include_directories(SYSTEM velox/external/duckdb)
 include_directories(SYSTEM velox/external/duckdb/tpch/dbgen/include)
 
+# these were previously vendored in third-party/
+if(NOT VELOX_DISABLE_GOOGLETEST)
+  set(gtest_SOURCE BUNDLED)
+  resolve_dependency(gtest)
+  set(VELOX_GTEST_INCUDE_DIR
+      "${gtest_SOURCE_DIR}/googletest/include"
+      PARENT_SCOPE)
+endif()
+
+set(xsimd_SOURCE BUNDLED)
```

Comment:
Hi @assignUser, may I know the reason for must use BUNDLED xsimd?

Reply:
Hi, xsimd was previously build from a fixed commit submodule so when converting this to resolve_dependency I set it this way to mirror this. I do not know if there are any technicalreasons for that specific xsimd version, it might be possible to use system xsimd? @pedroerp?

Reply:
@Yuhta was the one working with xsmid. Do you remember the reason?